release-scripts/do_release.py

```diff
@@ -91,6 +91,18 @@
     (re.compile(r'__version__\s*=\s*".*"'),
      '__version__ = "{version-no-v}-dev"'),
 
+    (re.compile(r'__libnetwork_plugin_version__\s*=\s*".*"'),
```

Comment:
You are quite right - I'd mis-remembered how the libnetwork and calico-node Docker images were specified in the doc string - I *thought* that it uses the versions in the __init__ file, but actually for those two it doesn't - they are hard coded in master to be  "latest".

Yeah I was thinking -dev might be sensible too.   Just need to make sure we handle that correctly when reading it in at start of day.